src/main/java/com/owncloud/android/lib/common/SearchResultEntry.kt

```diff
@@ -54,17 +59,20 @@ data class SearchResultEntry(
     }
 
     private fun parseRemotePath(): String {
-        val sanitizer = UrlQuerySanitizer()
-        sanitizer.allowUnregisteredParamaters = true
-        sanitizer.unregisteredParameterValueSanitizer = UrlQuerySanitizer.getAllButNulLegal()
+        val sanitizer = UrlQuerySanitizer().apply {
+            allowUnregisteredParamaters = true
+            unregisteredParameterValueSanitizer = UrlQuerySanitizer.getAllButNulLegal()
+        }
+
         sanitizer.parseUrl(resourceUrl)
-        URL(resourceUrl).query?.let { sanitizer.parseQuery(it) }
```

Comment:
We don't need to do this. `parseUrl` already calls `parseQuery` if present. Additionally this crashes when trying to parse paths that are not valid URLs (like files paths without the server).